database/select.py

```diff
@@ -0,0 +1,27 @@
+from time import sleep
+
+import pymysql
+import pymysql.cursors
+
+from database.connection import db_connect
+
+def db_select(db):
+    cursor=db.cursor()
+
+
+    PostgreSQL_select_Query = """select column_name from information_schema.columns where table_name = 'tbl_employee1' """
```

Comment:
Remove code of line 12 to 15. I don't think we are using it anywhere.

Reply:
For just reference of columns while running  code to match all inserted data are correct with columns